content/manuals/engine/logging/drivers/awslogs.md

```diff
@@ -127,6 +127,30 @@ $ docker run \
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before
 > you attempt to use `awslogs-create-group`.
 
+### awslogs-create-stream
+
+By default, the log driver will attempt to create the AWS CloudWatch Logs stream to be used
+for container log persistence.
+
+To disable AWS CloudWatch Logs stream creation, set `awslogs-create-stream` to `false`.
+
+The `awslogs-create-stream` option defaults to `true`.
+
+```console
+$ docker run \
+    --log-driver=awslogs \
+    --log-opt awslogs-region=us-east-1 \
+    --log-opt awslogs-group=myLogGroup \
+    --log-opt awslogs-stream=myLogStream \
+    --log-opt awslogs-create-stream=false \
+    ...
+```
+
+> [!NOTE]
+>
+> By disabling log stream creation, the Docker daemon assumes the log stream configured via
+> `awslogs-stream` option or a log stream of the container ID (default) already exists.
+
```

Comment:
What happens in situations where the option is disabled, and the logstream doesn't exist? Does it produce an error? (and if so; perhaps slightly less relevant; does the error get returned when _creating_ or _starting_ the container?)

I was slightly pondering if we could somehow roll this note up into the text above (trying to avoid too many "notes"), but not sure if that's a good option;

> Set awslogs-create-stream to false to disable AWS CloudWatch Logs stream creation. If
> this option is disabled, the Docker daemon assumes the log stream configured ....

Reply:
Yeah my intent was to shout-out this could be a foot gun for folks if there is no other mechanism handling log stream creation. Let me to try to find the error message that will appear in Docker logs so we can be explicit in the messaging.